CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ Release date: TBD
 
 ### Bug Fixes
 
+#### All platforms
+- An existing application instance will be reused instead of starting another instance.
```

Comment:
In my feel, this PR is improvement. The behavior is normal for almost applications whether it’s good or bad. Just existing multiple instances at once.